tst/Erg/ergtabletestsuite.cpp

```diff
@@ -0,0 +1,62 @@
+#include "ergtabletestsuite.h"
+
+#include <QCoreApplication>
+
+#include "Tools/testsettings.h"
+
+
+ErgTableTestSuite::ErgTableTestSuite()
+{
+
+}
+
+
+
+void ErgTableTestSuite::test_wattageEstimation(const QList<DataPoint> &inputs, const QList<DataPoint>& expectedOutputs) {
+
+    TestSettings testSettings("Roberto Viola", "QDomyos-Zwift Testing");
+    testSettings.activate(); // previous settings should be restored automatically when this object goes out of scope via its destructor
+
+    // "just in case", clear the DataPoints settings so that the ergtable is blank
+    testSettings.qsettings.remove("DataPoints");
```

Comment:
This bit here is not ideal IMO. 
The TestSettings class is something I made to manage a QSettings profile for a test case. There's invisible cleanup happening here. I think it would be better if the ergTable's automatic saving and loading could be optional (and turned off in these tests).

Reply:
sure it makes sense.

Reply:
i changed this in order to reuse our standard qzsettings